src/Symfony/Component/DependencyInjection/Compiler/PriorityTaggedServiceTrait.php

```diff
@@ -76,6 +76,8 @@ private function findAndSortTaggedServices($tagName, ContainerBuilder $container
 
                 if (null !== $indexAttribute && isset($attribute[$indexAttribute])) {
                     $index = $attribute[$indexAttribute];
+                } elseif (isset($attribute['name'])) {
+                    $index = $attribute['name'];
```

Comment:
What's this for? Is it to correspond with the new code in ResolveInstanceof...? If so, does `index` bit better here (I know we now have some inconsistency between index and name...)